src/views/CreateOrUpdateRequest.svelte

```diff
@@ -129,15 +123,41 @@ async function cancelRequest() {
               </span>
             </div>
 
-            <GooglePlacesAutocomplete bind:value={request.destination} placeholder="Where?" {options} apiKey={process.env.GOOGLE_PLACES_API_KEY} styleClass="form-control form-control-lg" />
+            <LocationInput class="form-control form-control-lg" on:change={onDestinationChanged} placeholder="Destination city" />
           </div>
         </div>
       {:else}
-        <!-- TODO: need to learn how to preload the GPA with existing values while having the value get loaded with the right location object, for now this is readonly  -->
+        <!-- TODO: Need to figure out how to make the locations editable. For now this is readonly. -->
```

Comment:
you can remove this altogether, we'll definitely came back to this when the feature gets brought up again, no need for the reminder here anymore.